src/app/plugins/quote.plugin.ts

```diff
@@ -0,0 +1,93 @@
+import Constants from '../../common/constants';
+import { Plugin } from '../../common/plugin';
+import { ChannelType, IContainer, IHttpResponse, IMessage } from '../../common/types';
+import { MessageEmbed } from 'discord.js';
+
+export default class QuotePlugin extends Plugin {
+  public commandName: string = 'quote';
+  public name: string = 'Quote Plugin';
+  public description: string = 'Prints a random quote';
+  public usage: string = 'quote <tag (optional)>';
+  public override pluginAlias = [];
+  public permission: ChannelType = ChannelType.Public;
+  public override pluginChannelName: string = Constants.Channels.Public.General;
+
+  private _DEFAULT_URL: string = 'https://api.quotable.io/random';
+  private _VALID_TAGS: string[] = [
+    'business',
+    'education',
+    'faith',
+    'famous-quotes',
+    'friendship',
+    'future',
+    'happiness',
+    'history',
+    'inspirational',
+    'life',
+    'literature',
+    'love',
+    'nature',
+    'politics',
+    'proverb',
+    'religion',
+    'science',
+    'success',
+    'technology',
+    'wisdom',
+  ];
+
+  constructor(public container: IContainer) {
+    super();
+  }
+
+  private _createEmbed(content: string, author: string, tag: string) {
+    const embed: MessageEmbed = new MessageEmbed()
+      .setColor('#0099ff')
+      .setTitle(`${tag}`)
+      .setFooter('Pulled from api.quotable.io')
+      .setTimestamp(new Date())
+      .setDescription(
+        `${content}
+
+        - ${author}`

```

Comment:
I dont think blank spaces work in embeds

Reply:
It works for me